tasks/recover/add_fcp_domain.yml

```diff
@@ -12,3 +12,14 @@
       auth: "{{ ovirt_auth }}"
       data_center: "{{ fcp_storage['dr_' + dr_target_host + '_dc_name'] }}"
       fcp: {}
+  register: result
+
+- name: Append to succeed_storage_domains
```

Comment:
Please rename to Log append to succeed_storage_domains or something like this, so it will be clear it is for logging

Reply:
Done